corehq/apps/app_manager/tasks.py

```diff
@@ -30,7 +30,7 @@ def make_async_build(app, username, release=False, comment=None):
             previous_version=latest_build,
             comment=comment,
         )
-        copy.is_auto_generated = True
+        copy.is_auto_generated = not release
```

Comment:
This seems odd. Based on the comment above, won't it be saved anyway if it's a released app?

Reply:
The `release` and `comment` params are always provided together, and they're provided in the one instance when this function *shouldn't* mark the app to be pruned: when appcues [makes a sample app for new users](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/views/apps.py#L455). That's not at all clear from this code. Added 9dae3bc69b2 - does that make this read more logically?